PusherClient/Pusher.cs

```diff
@@ -130,7 +130,7 @@ public Channel Subscribe(string channelName)
             if (_connection.State != ConnectionState.Connected)
                 throw new PusherException("You must wait for Pusher to connect before you can subscribe to a channel", ErrorCodes.NotConnected);

```

Comment:
With the change [here](https://github.com/pusher-community/pusher-websocket-dotnet/pull/9/files#diff-25180fb73bd9f8ee4a11d4b830a8bec4R261) it would seem that the `PusherException` is no longer required.